htdocs/js/ui/discovery_page.js

```diff
@@ -35,7 +35,7 @@ RCloud.UI.discovery_page = (function() {
                                     last_commit: new Date(current.last_commit).toDateString(),
                                     username: current.username,
                                     num_stars: editor.num_stars(current[0]),
-                                    image_src: "data:image/png;base64," + thumb_src,
+                                    image_src: thumb_src,
```

Comment:
hmm, kind of looks like you didn't test this, since the base64 needs to be added back somewhere. i'll fix this.